.github/workflows/ci.yml

```diff
@@ -12,16 +12,30 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Init Hermit
-        run: ./bin/hermit env -r >> $GITHUB_ENV
+        run: ./bin/hermit env -r >> "$GITHUB_ENV"
```

Comment:
Quoting here was suggested by staticcheck.
I can remove it if you'd prefer not to have that in this change.